### DISCUSSION TITLE
Prefix argument on on git-commit-review to choose from any already known contributors

### DISCUSSION THREAD

**Discussion by apteryks** [magit/magit#4998] [Ideas]
Hello,

In some project such as GNU Guix, we submit our own patches after they were  reviewed by others, which introduces a need to mark the reviewer as someone else; currently the `git-commit-review` procedure completes with my own name <email>.

This proposition would be to allow doing something like: `C-u C-c C-r` to let the author choose any known contributors in the Git history, like what `git-commit-suggested` does.

I can also see having this ability to add `Tested-by:` tags useful.

Thoughts?

**Comment by tarsius** [comment]
Implemented in 186414ae418a07a46c413f05c68413a76256a05e.

Note that for updated information in the transient buffer, you need https://github.com/magit/transient/commit/cc0fa80530b02493f73b870032bfcdd1435286cd. I meant to do that for a long time, and this gave me an opportunity to try it out in a simple case.

Simple is relative though, we could change the displayed information more drastically when a universal argument is in effect, here and elsewhere. But that's a task for another day.

**Reply by apteryks** [reply]
Wow, that was fast :-)  Thank you, Jonas!

**Reply by tarsius** [reply]
Cheers!
